benchmark/src/renderer/svg2img.ts

```diff
@@ -0,0 +1,10 @@
+import svg2img from 'svg2img';
+import { Render } from './types';
+
+export const render: Render = (svg) =>
+  new Promise((resolve, reject) => {
+    svg2img(svg, (err, buf: Buffer) => {
```

Comment:
Here it looks like the width/height parameters are missing, causing the generated PNG to be cropped.

Reply:
@fuzhenn PTAL.